src/contact/contact.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { ReturnModelType } from '@typegoose/typegoose';
+import { InjectModel } from 'nestjs-typegoose';
+import { Contact } from './contact.model';
```

Comment:
Please, add blank line after node_modules definitions and before project files definitions